Preprocessing/BOT_MAKE_TRANSCRIPTION.py

```diff
@@ -1,42 +1,44 @@
 import argparse
 import os
+from pathlib import Path
 import subprocess
 
 
-def przetworz_folder(folder_wejsciowy: str, folder_wyjsciowy: str) -> None:
-    model = "large-v3"
-    jezyk = "Polish"
-    urzadzenie = "cuda"
+def process_audio_files(input_folder: str, output_folder: str, model: str = "large-v3", language: str = "Polish", device: str = "cuda") -> None:
```

Comment:
to samo co wczesniej, brak domyslnych argumentow funkcji, domyslne argparsera jako const static skryptu